### ISSUE TITLE
sigaltstack: Cannot allocate memory

### ISSUE THREAD

**Issue by saraedum** [sagemath/cysignals#69]
Trying to package the latest release for conda, we get the following error on OSX:
```
export PYTHONPATH="`pwd`/tmp/site-packages" && python -B rundoctests.py src/cysignals/*.pyx
Doctesting 5 files.
sigaltstack: Cannot allocate memory
```
Full details can be found here https://travis-ci.org/conda-forge/cysignals-feedstock/jobs/348857291.

**Comment by saraedum** [comment]
Sorry, duplicate of #68.

**Comment by jdemeyer** [comment]
Fully fixed in version 1.6.8
